### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by matzew**
* moving mock-data-loader one level up
* in `pom.xml`, using latest jboss-parent
* gitignore changes

@ziccardi @josemigallas  please review/merge

### CODE REVIEW

**Comment by matzew** [comment]
@ziccardi I think we should have a README on the `mock-data-loader` folder about usage

**Comment by matzew** [comment]
@ziccardi lol - nevermind, just saw it, I think I lost it during strucutre changes :smiley_cat:

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Some tools and scripts for a UPS test suite
 
 ## Mocked Data
 
-A tool to produce mock data (push applications, variants and tokens) can be found into the [ups-test-suite/mock-data-loader](ups-test-suite/mock-data-loader) folder.
+A tool to produce mock data (push applications, variants and tokens) can be found into the [mock-data-loader](mock-data-loader) folder.
```
Comment (josemigallas) [inline README.md]: I would include this section below _Automated Tools_, with the other tools, and change _Automated Tools_ into plain _Tools_.

**Review by josemigallas — CHANGES_REQUESTED** [review]
Looks good to me, except for the only comment about readme.

**Comment by matzew** [comment]
@josemigallas done!

**Review by josemigallas — APPROVED** [review]

**Pull request merged**